tools/meteor-services/service-connection.js

```diff
@@ -50,6 +51,11 @@ var ServiceConnection = function (endpointUrl, options) {
       connectFuture.return();
     }
   });
+  if (process.env.CAFILE) {
+    options.npmFayeOptions = {
+      ca: fs.readFileSync(process.env.CAFILE)
```

Comment:
It seems files is undefined in this context. Do you have a suggestion as to where or how I can get a reference to files here? Should I add this code instead `var files = require('../fs/files.js');`